CHANGELOG.md

```diff
@@ -15,6 +15,19 @@
   * Add `BTThreeDSecureError.deallocated` = 8
 * BraintreeSEPADirectDebit
   * Add `BTSEPADirectDebitError.deallocated` = 8
+* BraintreeVenmo
+  * Allow merchants to collect enriched customer data if enabled in the Braintree Control Panel
+  * Add the following properties to `BTVenmoRequest`
+    * `collectCustomerBillingAddress`
+    * `collectCustomerShippingAddress`
+    * `totalAmount`
+    * `subTotalAmount`
+    * `discountAmount`
+    * `taxAmount`
+    * `shippingAmount`
+    * `lineItems`
+  * Add new `BTVenmoError` 
+    * `.enrichedCustomerDataDisabled`
```

Comment:
We ended up favoring having all of our error code updates be in this[ SDK_ERROR_CODES.md file](https://github.com/braintree/braintree_ios/blob/master/SDK_ERROR_CODES.md) instead of the CHANGELOG.

I can make that update for you, but don't have push access to your fork.

Reply:
Thanks @scannillo. I'm noticing that the Error file does not have any `BTVenmoError`s listed under the `BraintreeVenmo` section. There's one `BTVenmoAppSwitchError` but it's missing some other existing BTVenmoError codes listed [here](https://github.com/khushboo18/braintree_ios/blob/c5500d05b69b0bb7b371161c7f1ff35be1f5c153/Sources/BraintreeVenmo/BTVenmoError.swift#L29). Are those left out on accident?

Reply:
Nope, we just missed those, not intentionally left off, they should absolutely be on there. 🤦

Reply:
Sounds good! I copied over those changes to this PR as well in case we want to merge it all together. If not, will rebase once your PR goes through @jaxdesmarais

Reply:
Merged in the other PR earlier today

Reply:
Rebased on the latest master :)